src/Core/Prefab/Globe/GlobeLayer.js

```diff
@@ -68,8 +66,6 @@ class GlobeLayer extends TiledGeometryLayer {
         this.options.defaultPickingRadius = 5;
         this.minSubdivisionLevel = this.minSubdivisionLevel == undefined ? 2 : this.minSubdivisionLevel;
         this.maxSubdivisionLevel = this.maxSubdivisionLevel == undefined ? 19 : this.maxSubdivisionLevel;
-        this.maxDeltaElevationLevel = this.maxDeltaElevationLevel || 4.0;
```

Comment:
Could we add a depreciation notice for this parameter?

Reply:
Should i leave the parameter and add "deprecated" instead of removing it ?

Reply:
Nope, you should indeed remove it.